piccolo_admin/endpoints.py

```diff
@@ -364,6 +366,28 @@ def __init__(
             response_model=UserResponseModel,
         )
 
+        api_app.add_route(
+            path="/change-password/",
+            route=change_password(
+                login_url="./../../auth/login/", session_table=session_table
+            ),
+            methods=["POST"],
+        )
+
+        api_app.mount(
+            path="/register/",
+            app=AuthenticationMiddleware(
+                register(
+                    redirect_to="./../../auth/login/", auth_table=auth_table
+                ),
+                SessionsAuthBackend(
+                    auth_table=auth_table,
+                    session_table=session_table,
+                    superuser_only=True,
+                ),
+            ),
+        )
```

Comment:
We need to be careful with these, because `create_admin` has a `read_only` parameter. We need a way of making these read only too. We can either modify the `register` and `change_password` endpoints to accept a `read_only` parameter, or return a generic endpoint which returns a 405 status code.

Reply:
I've added a PR to adding read only modes to these endpoints:

https://github.com/piccolo-orm/piccolo_api/issues/156

Reply:
Great. Thanks